backends/magma/ceed-magma-gemm-nontensor.cpp

```diff
@@ -5,7 +5,9 @@
 //
 // This file is part of CEED:  http://github.com/ceed
 
-#include "ceed-magma.h"
+#include "ceed-magma-gemm-nontensor.h"
+
+#include "ceed-magma-gemm-selector.h"
```

Comment:
minor - did clang-format put that extra line break in there?

Reply:
No that was me following the style for header/definiton file pairs to include first the associated header with delcarations, followed by newline, then IWYU. The same approach is in `ceed-magma-gemm-selector.cpp`. Should I change it to group all local includes together in the same block?

Reply:
I wasn't aware of such a convention - it doesn't look like we uniformly use it. I have no strong feelings